### PR TITLE
add git helper to installer build

### PR DESCRIPTION
**Pull request by zanderz**
also delete some old scripts.
Built + tested.
@keybase/react-hackers

### CODE REVIEW

File: packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs [hunk 1]
```diff
@@ -108,6 +108,19 @@
...
         <File Id='KbfsDokanExe' DiskId='1' Source="$(env.GOPATH)\src\github.com\keybase\kbfs\kbfsdokan\kbfsdokan.exe" />
       </Component>
+			
+      <Component Id="Path" Guid="{71a0b7e4-d6ce-4438-b6b5-e937efb8be3a}">
+	 <Environment Id="PATH" Name="PATH" Value="[INSTALLFOLDER]" Permanent="no" Part="last" Action="set" System="no" />
+      </Component>
+			
+      <Component Id="GitHelperExe" Guid="{d8938a5b-c45f-4942-b552-a357a5563329}">
+        <CreateFolder />
+        <RemoveFolder Id="RemoveDokan" On="uninstall" />
```
Comment (chrisnojima) [inline packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs]: should this be removedokan?

Comment (zanderz) [reply packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs]: yes, thanks for catching that.

**Comment by zanderz** [comment]
it works! Successfully built + tested https://ci.keyba.se/job/windows-installer/lastSuccessfulBuild/artifact/src/github.com/keybase/client/packaging/windows/jenkins-windows-installer-1073/Keybase_1.0.32-20170928125708+226bd28.386.exe

**Comment by chrisnojima** [comment]
I'd test various git apps to see if this works
- [ ] github
- [ ] git bash
- [ ] others?

**Comment by chrisnojima** [comment]
code seems fine but we should test the various clients

**Comment by chrisnojima** [comment]
Didn't work with the git shell that comes w/ github for me

**Comment by chrisnojima** [comment]
git bash doens't work for me either

**Pull request merged**